packages/hash/frontend/src/pages/[account-slug]/types/entity-type/[entity-type-id].page/shared/entity-type-context.tsx

```diff
@@ -0,0 +1,14 @@
+import { EntityType } from "@blockprotocol/type-system";
+import { createContext, useContext } from "react";
+
+export const EntityTypeContext = createContext<null | EntityType>(null);
+
+export const useEntityType = () => {
```

Comment:
(optional): I prefer adding explicit return types for exported functions. This helps us avoid unintended breaking changes when we refactor their internals. TypeScript is also a bit faster in this case.